plugins/modules/proxysql_replication_hostgroups.py

```diff
@@ -278,17 +283,60 @@ def delete_repl_group(self, check_mode, result, cursor):
                              " mysql_replication_hostgroups, however" +
                              " check_mode is enabled.")
 
+    def update_check_type(self, cursor):
+        try:
+            query_string = \
+                """UPDATE mysql_replication_hostgroups
+                    SET check_type = %s
+                    WHERE writer_hostgroup = %s
+                """
+
+            query_data = \
+                [self.check_type, self.writer_hostgroup]
+
+            cursor.execute(query_string, query_data)
+        except Exception as e:
+            pass
+
+    def update_reader_hostgroup(self, cursor):
+        query_string = \
+            """UPDATE mysql_replication_hostgroups
+                SET reader_hostgroup = %s
+                WHERE writer_hostgroup = %s
+            """
+
+        query_data = \
+            [self.reader_hostgroup, self.writer_hostgroup]
+
+        cursor.execute(query_string, query_data)
+
+    def update_comment(self, cursor):
+        query_string = \
+            """UPDATE mysql_replication_hostgroups
+                SET comment = %s
+                WHERE writer_hostgroup = %s
+            """
+
+        query_data = \
+            [self.comment, self.writer_hostgroup]
+
+        cursor.execute(query_string, query_data)
+
+
 # ===========================================
 # Module execution.
 #
-
-
 def main():
     argument_spec = proxysql_common_argument_spec()
     argument_spec.update(
         writer_hostgroup=dict(required=True, type='int'),
         reader_hostgroup=dict(required=True, type='int'),
-        comment=dict(type='str'),
+        check_type=dict(type='str', default='read_only', choices=["read_only",
+                                                                  "innodb_read_only",
+                                                                  "super_read_only",
+                                                                  "read_only|innodb_read_only",
+                                                                  "read_only&innodb_read_only"]),
```

Comment:
```suggestion
        check_type=dict(type='str', default='read_only', choices=['read_only',
                                                                  'innodb_read_only',
                                                                  'super_read_only',
                                                                  'read_only|innodb_read_only',
                                                                  'read_only&innodb_read_only']),
```
i would suggest using a common quote style in adjacent pieces of code